xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-default/src/main/java/org/xwiki/mentions/internal/DefaultMentionXDOMService.java

```diff
@@ -63,8 +65,7 @@
     private Logger logger;
 
     @Inject
-    @Named("xwiki/2.1")
-    private Parser parser;
+    private ComponentManager componentManager;
```

Comment:
I'm wondering if you shouldn't use https://extensions.xwiki.org/xwiki/bin/view/Extension/Component%20Module#HContextComponentManager instead.